content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/linux.md

```diff
@@ -63,8 +63,8 @@ To customize tracer versions:
    ```
 
 1. Find your language(s) and use the dropdown to either:
-   - Pin an exact tracer version, or
-   - Select the major version you want to use.
+   - Select an exact SDK version, or
+   - Select the major version which will use the latest minor release of the selected major version at the time the Agent install command is run.
```

Comment:
```suggestion
   - Select the major version you want to use. The system installs the latest minor release of the selected major version.
```

Reply:
Can you reword line 67 to be present tense, to conform to Datadog's style guide?

Reply:
I've updated the tense - I wasn't sure about the rule since I was trying to convey something that will happen in the future (at the time the command is run).